src/Orangehill/Iseed/Iseed.php

```diff
@@ -61,8 +61,11 @@ public function readStubFile($file)
      */
     public function generateSeed($table, $database = null, $max = 0, $exclude = null, $prerunEvent = null, $postrunEvent = null, $dumpAuto = true, $indexed = true, $orderBy = null, $direction = 'ASC')
     {
+        $isDefaultDatabase = true;
+
         if (!$database) {
             $database = config('database.default');
+            $isDefaultDatabase = false;
```

Comment:
Hey @joyce-indzz I finally have some free time to go through the PRs :) Don't you think this is upside down? If no $database provided, shouldn/t $isDefaultDatabase be true? Please check and update PR.